chatbot-configuration/load_data.py

```diff
@@ -0,0 +1,241 @@
+#!/usr/bin/env python
+
+import argparse
+import dialogflow
+import json
+import logging
+
+
+def create_entity_type(project_id, display_name, kind=dialogflow.enums.EntityType.Kind.KIND_MAP):
+    entity_types_client = dialogflow.EntityTypesClient()
+
+    parent = entity_types_client.project_agent_path(project_id)
+    entity_type = dialogflow.types.EntityType(
+        display_name=display_name, kind=kind)
+
+    response = entity_types_client.create_entity_type(parent, entity_type)
+    logger.debug('Entity type created: \n{}'.format(response))
+
+
+def _get_entity_type_ids(project_id, display_name):
+    entity_types_client = dialogflow.EntityTypesClient()
+
+    parent = entity_types_client.project_agent_path(project_id)
+    entity_types = entity_types_client.list_entity_types(parent)
+    entity_type_names = [
+        entity_type.name for entity_type in entity_types
+        if entity_type.display_name == display_name]
+
+    entity_type_ids = [
+        entity_type_name.split('/')[-1] for entity_type_name
+        in entity_type_names]
+
+    return entity_type_ids
+
+
+def create_or_update_entities(project_id, entity_type_id, entities_json):
+    entity_types_client = dialogflow.EntityTypesClient()
+    entity_type_path = entity_types_client.entity_type_path(
+        project_id, entity_type_id)
+
+    entities = []
+    for entity_json in entities_json:
+        # Note: synonyms must be exactly [entity_value] if the
+        # entity_type's kind is KIND_LIST
+        synonyms = entity_json['synonyms'] or [entity_json['value']]
+
+        entity = dialogflow.types.EntityType.Entity()
+        entity.value = entity_json['value']
+        entity.synonyms.extend(synonyms)
+
+        entities.append(entity)
+
+    response = entity_types_client.batch_update_entities(entity_type_path, entities)
+    logger.debug('Entities batch created or updated: \n{}'.format(response))
+
+
+def _get_message_from_json(intent):
+    messages = []
+    for message in intent['messages']:
+        messages.append(message['text'])
+
+    text = dialogflow.types.Intent.Message.Text(text=messages)
+    message = dialogflow.types.Intent.Message(text=text)
+    return message
+
+
+def _get_training_phrases_from_json(intent):
+    training_phrases = []
+    if 'training_phrases' in intent:
+        for training_phrase in intent['training_phrases']:
```

Comment:
An alternative for previous comment could be:
```python
for training_phrase in intent.get('training_phrases', []):
```

This kind of improvements (`if` or `for` ones) can be done at some other places, please, take a look if you think the improvement makes sense.